src/components/Subtitle.tsx

```diff
@@ -0,0 +1,11 @@
+function Subtitle() {
+  return (
+    <p className="subtitle">
+      sparkboxers review food and
+      <br />
+      drink places so you can eat right
```

Comment:
I think we should change the wording here to be more general since the audience is no longer only people at Sparkbox.

```suggestion
      Sparkeats restaurant reviews
      <br />
      so you can eat right
```

Reply:
Updated in d97c677e9a8d528d0e7ba6ef17b3578feb1568c5.